website/docs/guides/qwik.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Qwik
+---
+
+# Qwik
+
+To use Fontsource with a [QwikCity](https://qwik.builder.io) project, it is suggested to use the `/routes/layout.tsx` file.
+
+<CarbonAd />
+
+## Installation
+
+---
+
+To start, install the fonts you want using your package manager of choice.
+
+```
+npm install @fontsource/abeezee
+yarn add @fontsource/abeezee
+pnpm install @fontsource/abeezee
+```
+
+Then in the `routes/layout.tsx` file, simply import those weights you need (or if variable, you can import the variable).
```

Comment:
Package installation is not needed here as that's already covered in "Getting Started".

```suggestion
In the `routes/layout.tsx` file, simply import those weights you need (or if variable, you can import the variable).
```